rightmove_webscraper/__init__.py

```diff
@@ -1,11 +1,54 @@
 
+import ast
+import re
 import datetime as dt
+from datetime import datetime
 from lxml import html
 import numpy as np
 import pandas as pd
 import requests
+from bs4 import BeautifulSoup
+import json
```

Comment:
Minor nit: you can follow PEP8 import orders (up to you)
```python
import ast
import datetime as dt
import json
import re

import numpy as np
import pandas as pd
import requests
from bs4 import BeautifulSoup
from lxml import html
```

Reply:
oh yes, actually I need to clean that import list - I was quickly prototyping the changes :)